db/migrate/20210929150807_remove_currency_from_gemini_connections.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveCurrencyFromGeminiConnections < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :gemini_connections, :default_currency
```

Comment:
Can you make sure this won't break anything? If it's safer to leave, then we can leave it.

Reply:
Actually I saw the method definition. LG2M